tests/test__mongomock.py

```diff
@@ -216,6 +217,31 @@ def client(uri):
         with self.assertRaises(ConfigurationError):
             c.get_default_database()
 
+    def test__getting_default_database_with_default_parameter(self):
+        def client(uri):
```

Comment:
No need to create that here as you ony create one client.

Same below

Reply:
You're right. Sorry for the cargo-cult.

I've dropped them and rebased the PR.